web-server/src/orm/entities/tasks/Task.ts

```diff
@@ -12,7 +12,7 @@ export class Task {
   id: number;
 
   @ManyToOne(() => User)
-  ownerID: User;
+  ownerId: User;
```

Comment:
No. You want this to be `owner: User`. Then you add a SECOND property called `ownerId: number`.